lib/cli.js

```diff
@@ -1754,10 +1776,12 @@ CLI.prototype.do_import.description = (
     'Options:\n' +
     '    -S SOURCE-URL      URL of the remote IMGAPI repository source.\n' +
     '                       This is the full URL to the origin IMGAPI\n' +
-    '                       without any suffix endpoint or action. Example:\n' +
-    '                       https://images.joyent.com. To import from a\n' +
-    '                       particular source *channel*, append\n' +
-    '                       "?channel=CHANNEL" to the URL.\n' +
+    '                       without any suffix endpoint or action, e.g.:\n' +
+    '                       https://images.joyent.com.\n' +
+    '                       https://updates.joyent.com.\n' +
+    '\n' +
```

Comment:
Let's not have this extra blank line so that the -S and -C options are cuddled together, as they are related.